### DISCUSSION TITLE
Failed to record new usage data

### DISCUSSION THREAD

**Discussion by regsw** [jertel/vuegraf#14] [General]
Hi, 

Great work on the Grafana and InfluxDB python! I was able to get it running yesterday and it was recording data just fine. 

Today after I started the script, I keep getting the following error and no data is being recorded:

| ERROR | Failed to record new usage data: (<class 'requests.exceptions.HTTPError'>, HTTPError('403 Client Error: Forbidden for url: https://api.emporiaenergy.com/usage/devices?start=<DATE/TIME and CustomerGid stuff here) <traceback object at 0x7f55cdc14bc8>

Maybe they are limiting something or do I have to reset something?

Thank you!
Mike

**Comment by jertel** [comment]
Yes I noticed the problem as well, starting about 30 minutes ago. My guess is either Emporia is performing system updates on their cloud servers, or they have changed the API which has broken Vuegraf. @magico13 wrote the Python lib that interfaces with Emporia so I am hoping he can take a look at it if the problem persists.

**Reply by magico13** [reply]
I wrote up https://github.com/magico13/PyEmVue/issues/11 to look into this later today as it's affecting the Home Assistant integration as well. Fingers crossed it's something I can fix pretty quickly.

**Comment by jertel** [comment]
I'm adjusting vuegraf to use the updated pyEmVue lib. Will keep you posted with the results.

**Reply by regsw** [reply]
That's great, thanks to both you and @magico13 for figuring this out so soon!

**Reply by magico13** [reply]
There are a number of people still experiencing some issues with the updates so let me know if any of you have any problems.

**Reply by jertel** [reply]
I've got it updated and it's been running fine now for about 30 minutes. I appreciate your quick response to update the pyEmVue lib. Thanks!

**Comment by jertel** [comment]
I've released Vuegraf 1.1.0 which resolves this problem. The new jertel/vuegraf image is available on Docker Hub.

**Comment by regsw** [comment]
Thank you, this is now working for me once again! I originally thought I had a problem with the new Vuegraf 1.1.0 script but that was because I didn't do the updates in requirements.txt. Once I ran "pip3 install -r requirements.txt" and started up the script again it was working perfectly.

**Reply by pc-99** [reply]
Thanks, this turned on my lightbulb, I was running "pip install" and not "pip3 install" and it wasn't working, now it's all good.

**Reply by jertel** [reply]
Good feedback. I've updated the readme to recommend trying pip3/python3 if pip/python are still pointing to python2.
